internal/cert-helpers/cert-helpers.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2022 The Inspektor Gadget authors
```

Comment:
Any reason to have this as a different package. Maybe we can fusion `internal/certs` and `internal/cert-helpers` packages?